src/components/KeyboardShortcuts/index.tsx

```diff
@@ -126,6 +175,14 @@ export class KeyboardShortcutsBase extends React.Component<Props> {
             <kbd>j</kbd>
           </dt>
           <dd>{gettext('Down file')}</dd>
+          <dt>
+            <kbd>p</kbd>
+          </dt>
+          <dd>{gettext('Previous change')}</dd>
```

Comment:
I suggest rendering these in a disabled state when `compareInfo` is `null`. This would help communicate that they aren't applicable to the browse page.

Reply:
Good idea. I have created some styles for this. Let me know if they look good, or if you'd like to see them changed.